src/sql/workbench/contrib/notebook/browser/cellViews/linkHandler.directive.ts

```diff
@@ -41,7 +44,7 @@ export class LinkHandlerDirective {
 		try {
 			const href = target['href'];
 			if (href) {
-				this.handleLink(href);
+				Promise.resolve(this.handleLink(href));
```

Comment:
Do exceptions from handleLink still get surfaced here? The promise isn't being awaited, or handled via catch/then.

Reply:
You're also getting a hygiene build failure here due to unhandled promises.

Reply:
I'd suggest making this handler async and then await on the call to this.handleLink - that will let the try/catch work properly